src/App.css

```diff
@@ -29,3 +29,16 @@ html, body, #root,  .App {
     pointer-events: none;
     color: black;
 }
+
+.no-padding-bottom {
+    padding-bottom: 0 !important;
```

Comment:
can we remove the `!important` if that is not necessary?

Reply:
It is not working without adding `!important`.